lightnode/kvstore/store.go

```diff
@@ -0,0 +1,41 @@
+package kvstore
+
+import (
+	"errors"
+	"github.com/syndtr/goleveldb/leveldb/iterator"
+)
+
+// Store implements a key-value store. May be backed by a database like LevelDB.
+type Store interface {
```

Comment:
Note this interface and its levelDB impl exist in DA Node already (https://github.com/Layr-Labs/eigenda/blob/master/node/db.go)

Reply:
Yes, I copied this interface. The reason why this exists in isolation is to avoid making any changes to existing code as a part of this PR. 

In the future, I think these interfaces should be merged. Do you think they should be merged now, or can that wait for a later time?

Reply:
Change made. I didn't utilize the new implementation for the TTL feature, since in doing so there may be some very non-trivial changes needed. But I did make it so that they both use the same low level store interface and implementation.